pageserve/templates/base.html

```diff
@@ -32,44 +34,60 @@
 </head>
 
 <body>
+    <div class="topnav">
+        <div id="nav_left">
+            <a class="{% block home_tab_active %}{% endblock %} navtab navbtn" href='/v1/'>Home</a>
+            <a class="{% block events_tab_active %}{% endblock %} navtab navbtn" href='/v1/events'>Events</a>
+        </div>
+        <div id="user_nav">
+            {% block login_buttons %}
+            {% if not session['user'] %}
+            <div id="sign_in_button">
+                <div class="g-signin2" data-onsuccess="onSignIn" data-theme="dark"></div>
+                <script>
+                    function onSignIn(googleUser) {
+                        // Secure Google ID token to send to backend for verification.
+                        var id_token = googleUser.getAuthResponse().id_token;
+                        var xhr = new XMLHttpRequest();
+                        xhr.open('POST', "authenticate");
```

Comment:
It looks like this is a relative URL based on the page -- if you don't make this a reference like `/authenticate`, make sure that the page can only be served from `/` and not from a path like `/post` (where the data might be sent somewhere else).